src/auth/auth.service.ts

```diff
@@ -43,6 +44,8 @@ export class AuthService implements IAuthService {
     private jwtService: JwtService,
   ) {}
 
+  private readonly logger = new Logger(AuthService.name);
```

Comment:
We have to use the DI principle, remember? So, you shouldn't directly instantiate Logger

Reply:
https://docs.nestjs.com/techniques/logger#dependency-injection